src/services/lyrics.js

```diff
@@ -0,0 +1,10 @@
+const LYRICS_URL = 'https://api.lyrics.ovh/v1/';
+
+export const getLyrics = (artist, title) => {
+  return fetch(LYRICS_URL + `/${artist}/${title}`)
+    .then(res => {
+      if(!res.ok) throw 'Could not get lyrics';
```

Comment:
Excellent. you could also look for res.400 or res.404. This is probably the best way tho. Great idea!